Designer/ViewModel/ViewDesignViewModel.cs

```diff
@@ -195,6 +196,65 @@ public class ViewDesignViewModel : INotifyPropertyChanged {
             RenderRoom();
         }
 
+        /**
+         * Plaatst alle deuren en ramen die in de ruimte zitten
+         */
+        public void RenderRoomFrames()
+        {
+            if(Design.Room.RoomPlacements != null)
+            {
+                Debug.WriteLine(Design.Room.RoomPlacements.Count);
+                foreach (RoomPlacement frame in Design.Room.RoomPlacements)
+                {
+                    Polygon poly = frame.GetPoly().GetPolygon();
```

Comment:
wat is het nut van het overzetten als je uiteindelijk toch zelf weer de lijn toevoegd of poly

Reply:
goed punt, heb t aangepast 👍